.github/triage/jax_toolbox_triage/args.py

```diff
@@ -76,6 +84,15 @@ def parse_args():
             test case fails on this date.""",
         type=lambda s: datetime.date.fromisoformat(s),
     )
+    container_search_args.add_argument(
+        "--start-container",
```

Comment:
Rename to passing_container and failing_container to make it more clear which one must pass/fail?

Reply:
I was trying to keep it aligned with `--start-date` and `--end-date`, but I guess you're right it's clearer. Changed.